share/picongpu/examples/LaserWakefield/include/picongpu/param/fileOutput.param

```diff
@@ -0,0 +1,125 @@
+     *       note: only makes sense for partially ionized ions
+     *   - ChargeDensity: density * charge
+     *       note: for species that do not change their charge state, this is
+     *             the same as the density times a constant for the charge
+     *   - Energy: sum of kinetic particle energy per cell with respect to shape
+     *   - EnergyDensity: average kinetic particle energy per cell times the
+     *                    particle density
+     *       note: this is the same as the sum of kinetic particle energy
+     *             divided by a constant for the cell volume
+     *   - MomentumComponent: ratio between a selected momentum component and
+     *                        the absolute momentum with respect to shape
+     *   - LarmorPower: radiated Larmor power
+     *                  (species must contain the attribute `momentumPrev1`)
+     *
+     * for debugging:
+     *   - MidCurrentDensityComponent:
+     *       density * charge * velocity_component
+     *   - Counter: counts point like particles per cell
+     *   - MacroCounter: counts point like macro particles per cell
+     */
+    namespace deriveField = particles::particleToGrid;
+
+    /* ChargeDensity section */
+    using ChargeDensity_Seq = deriveField::CreateEligible_t<
+        VectorAllSpecies,
+        deriveField::derivedAttributes::ChargeDensity
+    >;
+
+    /* Density section */
+   using Density_Seq = deriveField::CreateEligible_t<
+        VectorAllSpecies,
+        deriveField::derivedAttributes::Density
+    >;
+
+    /* EnergyDensity section */
+    using EnergyDensity_Seq = deriveField::CreateEligible_t<
+        VectorAllSpecies,
+        deriveField::derivedAttributes::EnergyDensity
+    >;
+
+    /* MomentumComponentsection: define "component" as
+       0=X (default), 1=Y or 2=Z (results: [-1.:1.])
+     */
+    using MomentumComponent_Seq = deriveField::CreateEligible_t<
+        VectorAllSpecies,
+        deriveField::derivedAttributes::MomentumComponent< 0 >
+    >;
+
+    /** FieldTmpSolvers groups all solvers that create data for FieldTmp ******
+     *
+     * FieldTmpSolvers is used in @see FieldTmp to calculate the exchange size
+     */
+    using FieldTmpSolvers = MakeSeq_t<
+        Density_Seq,
+        ChargeDensity_Seq,
+        EnergyDensity_Seq,
+        MomentumComponent_Seq
+    >;
+
+
+    /** FileOutputFields: Groups all Fields that shall be dumped *************/
+
+    /** Possible native fields: FieldE, FieldB, FieldJ
+     */
+    using NativeFileOutputFields = MakeSeq_t<
+        FieldE,
+        FieldB
+    >;
+
+    using FileOutputFields = MakeSeq_t<
+        NativeFileOutputFields,
+        FieldTmpSolvers
+    >;
+
+
+    /** FileOutputParticles: Groups all Species that shall be dumped **********
+     *
+     * hint: to disable particle output set to
+     *   using FileOutputParticles = MakeSeq_t< >;
+     */
+    using FileOutputParticles = VectorAllSpecies;
+
+}
```

Comment:
Why is this file needed in the LWFA example? Does it differ from the default?

Reply:
This can be removed. I have added `Density` here but no need to use LWFA for saxs testing anymore.